pkg/recordset/stack_template.go

```diff
@@ -0,0 +1,128 @@
+package recordset
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	targetStackTemplate = `AWSTemplateFormatVersion: 2010-09-09
+Description: Recordset Guest CloudFormation stack.
+Resources:
+  ingressDNSRecord:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      HostedZoneId: {{ .HostedZoneID }}
+      Name: 'ingress.{{ .ClusterName }}.{{ .HostedZoneName }}'
+      Type: CNAME
+      TTL: '900'
+      ResourceRecords:
+      - {{ .IngressELBDNS }}
+
+  ingressWildcardDNSRecord:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      HostedZoneId: {{ .HostedZoneID }}
+      Name: '*.{{ .ClusterName }}.{{ .HostedZoneName }}'
+      Type: CNAME
+      TTL: '900'
+      ResourceRecords:
+      - {{ .IngressELBDNS }}
+
+  apiDNSRecord:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      HostedZoneId: {{ .HostedZoneID }}
+      Name: 'api.{{ .ClusterName }}.{{ .HostedZoneName }}'
+      Type: CNAME
+      TTL: '900'
+      ResourceRecords:
+      - {{ .APIELBDNS }}
+
+  etcdDNSRecord:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      HostedZoneId: {{ .HostedZoneID }}
+      Name: 'etcd.{{ .ClusterName }}.{{ .HostedZoneName }}'
+      Type: CNAME
+      TTL: '900'
+      ResourceRecords:
+      - {{ .EtcdInstanceDNS }}
+`
```

Comment:
This is unrelated but it would be good to align with the new delegation design.  I.e. creating separate `cluster-id.k8s` zone for every cluster. We can do that later when Route53 will be finally enabled in China, but let's keep that in mind.